src/api/server.ts

```
@@ -49,23 +53,67 @@ app.get(
+        responseTime: duration,
       });
     } catch (error) {
-      console.error("Error fetching leaderboard:", error);
+      logger.error("Error fetching leaderboard", error);
       res.status(500).json({ success: false, error: "Internal Server Error" });
     }
   },
 );
 
+// Cache analytics endpoint (for monitoring)
+app.get("/api/cache/analytics", securityLogger, async (_req: Request, res: Response) => {
```

ESLint

[error] Replace `"/api/cache/analytics",·securityLogger,` with `⏎··"/api/cache/analytics",⏎··securityLogger,⏎·`

(prettier/prettier)